tests/actions.rs

```diff
@@ -1,6 +1,5 @@
 //! Alert tests
-#[macro_use]
-extern crate serial_test;
+use serial_test::serial;
```

Comment:
I don't think there's a need for the empty line between this and the other `use` lines below any more.